src/app/core/services/kafka/kafka.service.ts

```diff
@@ -112,6 +121,24 @@ export class KafkaService {
       })
   }
 
+  processCacheValue(value: CacheValue | OldCacheValue): Promise<CacheValue> {
```

Comment:
@yatharthranjan Is it fine to support this?

Reply:
Yes, should be ok. Just seems a lot of custom code in the app for something so rare. Also please document this as it could be confusing for other contributors.

Reply:
Yeah I've just removed it since it adds custom code. They can just reset the cache when the other cache data has been sent.